great_expectations/data_context/data_context/context_factory.py

```diff
@@ -111,6 +112,9 @@ def get_datasources(self) -> DatasourceDict:
     def get_validator(self, batch_request: BatchRequest) -> Validator:
         return self._project.get_validator(batch_request=batch_request)
 
+    def get_suite_factory(self) -> SuiteFactory:
+        return self._project.suites
```

Comment:
i think our latest pattern here is to access the `__project` directly. but there's some inconsistency, so i'm not 100% sure.

Reply:
Hmm is it? I'm seeing a lot of `_project` usage since it is a property that will panic if the context isn't set?